addons/skin.estuary/xml/Variables.xml

```diff
@@ -82,6 +82,7 @@
 	<variable name="ListLabelVar">
 		<value condition="String.IsEqual(ListItem.DbType,episode) + Window.IsActive(videoplaylist)">$INFO[ListItem.TVShowtitle,,: ]$INFO[ListItem.Season,,x]$INFO[ListItem.Episode,,. ]$INFO[ListItem.Title]</value>
 		<value condition="String.IsEqual(ListItem.DbType,musicvideo) + Window.IsActive(videoplaylist)">$INFO[ListItem.Artist,, - ]$INFO[ListItem.Title]</value>
+		<value condition="String.IsEqual(ListItem.DbType,movie)">$INFO[ListItem.Label][I]$INFO[ListItem.VideoVersionName, (,)][/I]</value>
```

Comment:
If the condition is changed to `String.IsEqual(ListItem.DbType,videoversion` then the video version name will only be displayed when you're in the video version folder. Which makes sense as it's the only place it's useful.

Reply:
No, with latest changes this is no longer true. Technically, to get artwork and what not, the children are now of type movie.